python/xnvme-cy-bindings/auxiliary/autopxd_py.py

```diff
@@ -53,11 +53,14 @@ def get_definitions(pxd_f):
         _type, __func_name, func_name, arg_body = func.groups()
 
         args = []
-        for arg in re.finditer(
-            r"([_a-zA-Z0-9\*]+) ([_a-zA-Z0-9]+)(?:, )?", arg_body.strip("()")
```

Comment:
I think the best solution would be to add `(?:const )?` to the beginning of the regex, and keep status quo.